### PR TITLE
feat: Axis secondaryTick

### PR DESCRIPTION
**Pull request by wadjih-bencheikh18**
closes : https://github.com/zakodium/react-plot/issues/264

### CODE REVIEW

File: src/components/Axis/Ticks.tsx [hunk 1]
```diff
@@ -36,52 +40,40 @@ export function Ticks(props: Omit<TicksProps, 'children'>) {
   const {
     primaryTicks,
     getPositions,
-    // hiddenSecondaryTicks = false,
+    secondaryTickLength = 0,
```
Comment (targos) [inline src/components/Axis/Ticks.tsx:43]: Default value should be 2

**Review by targos — CHANGES_REQUESTED** [review]
There is something wrong with the implementation. If I put `secondaryTickLength={1}`, the length is the same as the primary ticks (which has a length of 5).

image

File: src/components/Axis/Ticks.tsx (hunk 1 above)
Comment (wadjih-bencheikh18) [reply src/components/Axis/Ticks.tsx]: fixed

**Review by targos — CHANGES_REQUESTED** [review]
https://deploy-preview-267--react-plot.netlify.app/?path=/story/api-parallelaxis--control

Parallel axis should have the same default.

**Comment by targos** [comment]
Maybe we should use 3 as the default value? With 2 it seems to short to me.

@lpatiny @stropitek wdyt?

**Comment by stropitek** [comment]
> Maybe we should use 3 as the default value? With 2 it seems to short to me.

Yes I had the same impression when I looked at the stories

**Comment by wadjih-bencheikh18** [comment]
default value changed

**Review by targos — APPROVED** [review]
LGTM as a first implementation. We can iterate in other issues/prs if necessary

**Pull request merged**